### PR TITLE
Add Cake.Jira.yml

### PR DESCRIPTION
**Pull request by Ninglin**

### CODE REVIEW

File: addins/Cake.Jira.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name: Cake.Jira
+NuGet: Cake.Jira
+Assemblies:
+- "/**/Cake.Jira.dll"
+Repository: https://github.com/Ninglin/Cake.Jira
+Author: Hugo Tavares
+Description: ""
```
Comment (pascalberger) [inline addins/Cake.Jira.yml]: Can you please add a description for the addin

Comment (Ninglin) [reply addins/Cake.Jira.yml]: I swear I wrote one 😂  must have gotten lost.

**Comment by Ninglin** [comment]
Anything missing so that we can merge this?

**Comment by pascalberger** [comment]
@Ninglin your changes have been merged, thanks for your contribution 👍

**Pull request merged**